kratos/python/add_processes_to_python.cpp

```diff
@@ -277,12 +277,14 @@ void  AddProcessesToPython(pybind11::module& m)
     ;
 
     py::class_<LevelSetConvectionProcess<2,SparseSpaceType,LocalSpaceType,LinearSolverType>, LevelSetConvectionProcess<2,SparseSpaceType,LocalSpaceType,LinearSolverType>::Pointer, Process>(m,"LevelSetConvectionProcess2D")
+        .def(py::init<Variable<double>&, Variable<array_1d<double, 3 > >&, ModelPart&, LinearSolverType::Pointer, const double, const double, const unsigned int, const bool, const bool>())
         .def(py::init<Variable<double>&, ModelPart&, LinearSolverType::Pointer>())
         .def(py::init<Variable<double>&, ModelPart&, LinearSolverType::Pointer, const double>())
         .def(py::init<Variable<double>&, ModelPart&, LinearSolverType::Pointer, const double, const double>())
         .def(py::init<Variable<double>&, ModelPart&, LinearSolverType::Pointer, const double, const double, const unsigned int>())
     ;
     py::class_<LevelSetConvectionProcess<3,SparseSpaceType,LocalSpaceType,LinearSolverType>, LevelSetConvectionProcess<3,SparseSpaceType,LocalSpaceType,LinearSolverType>::Pointer, Process>(m,"LevelSetConvectionProcess3D")
+        .def(py::init<Variable<double>&, Variable<array_1d<double, 3 > >&, ModelPart&, LinearSolverType::Pointer, const double, const double, const unsigned int, const bool, const bool>())
```

Comment:
Hm this is a lot of individual settings
I think a better solution would be a CTor that accepts `Parameters`

Reply:
Done.